app-client/pom.xml

```diff
@@ -42,9 +42,9 @@
 
         For more information about how to configure Maven for your application,
         see the section entitled 'Use the Maven Repository'
-        in the Development Guide for Red Hat JBoss Enterprise Application Platform 6 located here:
+        in the Development Guide for Red Hat JBoss Enterprise Application Platform located here:
 
-        https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Development_Guide/ 
```

Comment:
We should change it here too https://github.com/jboss-developer/maven-qstools-plugin/blob/master/config/qstools_config.yaml#L32 otherwise it will miss if we run qstools:repositories again